### PR TITLE
test/add list selection test case

### PR DESCRIPTION
**Pull request by JimmFly**
Used to test whether the list can be selected when the list icon is clicked
selection

Related to https://github.com/toeverything/blocksuite/pull/181

### CODE REVIEW

File: tests/selection.spec.ts [hunk 1]
```diff
@@ -347,3 +350,20 @@ test('select text leaving a few words in the last line and delete', async ({
   const textOne = await getQuillSelectionText(page);
   expect(textOne).toBe('abc89\n');
 });
+
+test('Click the list icon to select', async ({ page }) => {
+  await enterPlaygroundRoom(page);
+  await initEmptyState(page);
+  await initThreeList(page);
+  await assertRichTexts(page, ['123', '456', '789']);
+  await page.mouse.click(100, 165);
```
Comment (lawvs) [inline tests/selection.spec.ts:359]: Using coordinates directly is unreliable, can you change to use [locators](https://playwright.dev/docs/locators)?

**Review by lawvs — CHANGES_REQUESTED** [review]

File: tests/selection.spec.ts (hunk 1 above)
Comment (JimmFly) [reply tests/selection.spec.ts:359]: ok i will try to use it

File: tests/selection.spec.ts [hunk 2]
```diff
@@ -347,3 +350,20 @@ test('select text leaving a few words in the last line and delete', async ({
...
+  await copyByKeyboard(page);
+  await pasteByKeyboard(page);
+  await page.mouse.click(100, 230);
+  await copyByKeyboard(page);
+  await pasteByKeyboard(page);
+  await page.mouse.click(100, 290);
+  await page.keyboard.press('Backspace', { delay: 50 });
+  await page.mouse.click(100, 200);
+  await page.keyboard.press('Backspace', { delay: 50 });
```
Comment (lawvs) [inline tests/selection.spec.ts:368]: Please add some assertions to make sure the result is expected.

File: tests/utils/actions/misc.ts [hunk 3]
```diff
@@ -116,6 +116,10 @@ export async function initThreeList(page: Page) {
   await page.keyboard.press('Tab', { delay: 50 });
   await page.keyboard.type('789');
 }
+export async function getListIcon(page: Page, i = 0) {
+  const locator = page.locator('.affine-list-block__prefix').nth(i);
+  await locator.click();
+}
```
Comment (lawvs) [inline tests/utils/actions/misc.ts:122]: I don't think it is common enough to be added to the misc file.

**Review by lawvs — APPROVED** [review]

**Comment by lawvs** [comment]
@JimmFly Can you resolve the conflict?

**Pull request merged**